compress.go

```diff
@@ -44,13 +44,13 @@ type flusher interface {
 	Flush() error
 }
 
-func (w *compressResponseWriter) Flush() {
+func (cw *compressResponseWriter) Flush() {
 	// Flush compressed data if compressor supports it.
-	if f, ok := w.compressor.(flusher); ok {
-		f.Flush()
+	if f, ok := cw.compressor.(flusher); ok {
+		_ = f.Flush()
```

Comment:
We should check the error here

Reply:
How should we handle the error? We can't return the error as `compressResponseWriter.Flush()` doesn't returns an error. I can use the inbuilt log package to log it, but I have some concerns. Would the default inbuilt logger print the error in a different format or manner than the user's configured logger?

Reply:
Yeah, I guess there is no other error logging in this package either
@apoorvajagtap do you have any thoughts on this?